packages/app/src/app/pages/Sandbox/Editor/Content/Tabs/Tab/elements.ts

```diff
@@ -0,0 +1,88 @@
+import styled, { css } from 'styled-components';
+import CloseIcon from 'react-icons/lib/go/x';
+
+export const Container = styled.div`
```

Comment:
@Saeris You can define the props like this,  so you don't have to specify the `theme` type.

```ts
export const Container = styled.div<Props>`
  // ...
`;
```